dataGen.js

```diff
@@ -0,0 +1,96 @@
+const faker = require('faker');
+
+// generate an array of objects representing items
+// items also have reviews property which is a random array of reviews
+const createFakeData = () => {
+
+  let dataArray = [];
+  let samePrice = true;
+
+  // TODO: set to 10000000 once databases are figured out
+  for (let i = 0; i < 1000; i++) {
+    let reviews = [];
+
+    for (let j = 0; j < Math.floor(Math.random() * 15); j++) {
+      let easeOfAssembly = Math.floor(Math.random() * 5 + 1);
+      let valueForMoney = Math.floor(Math.random() * 5 + 1);

```

Comment:
curious about Math.random and faker's random number advantages or disadvantages.